application-admintools-ui/src/main/resources/AdminTools/Code/Translations.xml

```diff
@@ -39,6 +39,7 @@
 admin.admintools.description=Configure Admin Tools Application
 AdminTools.Code.ConfigurationClass_serverLocation.hint=Path to the server location
 AdminTools.Code.ConfigurationClass_serverLocation=Server location
+AdminTools.dashboard.section.title.backend=🌐**Backend **
```

Comment:
I cannot find the comment about Font Awesome. But as I said there, this icon should use that
The bold part can be done from html, no need to have it in the translation, since other translation would need to respect that

You could also create a new section in the translation page, since this is not related to the administration section, but to the dashboard page. And I suggest `adminTools.dashboard.section.backend.title` (title is the last one)